content/wiki/contests/BAPC2019.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "BAPC 2019"
+date: 2022-11-29T21:57:18+01:00
+menu: "wiki"
+---
+
+
+|            |                              |
+|------------|-----------------------------:|
+| Host       |  Radboud university Nijmegen |
+| Organizer  |             Desda and Thalia |
+| Committee  | [2019 BAPC Committee][email] |
+| BAPC Date  |       October {date}  {year} | 
+| Website    |              [link][website] |
+| Pictures   |               [link][photos] |
+| Scoreboard |           [link][scoreboard] |
+| Problemset |           [link][problemset] |
+| Teams      |                           56 |
+
+## Problemset
+| Contest     | Problemset                                                                         | Solutions                                                                         | Presentation                                                                      |
+|-------------|------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------|-----------------------------------------------------------------------------------|
+| BAPC        | [link](http://commissies.ch.tudelft.nl/chipcie/archive/{year}/bapc/problemset.pdf) | [link](http://commissies.ch.tudelft.nl/chipcie/archive/{year}/bapc/solutions.zip) | [link](http://commissies.ch.tudelft.nl/chipcie/archive/{year}/bapc/solutions.pdf) 
```

Comment:
this seems to be going wrong. needs a replace `{year}` by `2019`

Reply:
Yes indeed, this was copied from the resources which was apparently already incorrect. This has now been fixed.